storybookjs/storybook
# Perf: Lazy load OverlayScrollbars

Issue: -

## What I did

Use lazy loading for OverlayScrollbars.js so it gets split into a separate bundle.

## How to test

Verify that you can still properly scroll in the sidebar.

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
